src/java/org/apache/cassandra/db/ColumnFamilyStore.java

```diff
@@ -23,6 +23,7 @@
 import java.nio.ByteBuffer;
 import java.nio.file.Files;
 import java.util.*;
```

Comment:
nit: Were we still planning to expand the imports on all branches (or none)?

Reply:
Currently the imports are optimized in 3.0, I will propagate the same changes to the next branches
So as per the latest patch I didn't see a reason for more changes there.